packages/phone-number-privacy/combiner/src/tracing.ts

```diff
@@ -0,0 +1,46 @@
+import { JaegerExporter } from '@opentelemetry/exporter-jaeger'
+import { registerInstrumentations } from '@opentelemetry/instrumentation'
+import { Resource } from '@opentelemetry/resources'
+import { BatchSpanProcessor } from '@opentelemetry/sdk-trace-base'
+import { NodeTracerProvider } from '@opentelemetry/sdk-trace-node'
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions'
+
+import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node'
+
+const options = {
+  tags: [],
+  endpoint: process.env.TRACER_ENDPOINT,
+}
+
+// Optionally register instrumentation libraries
+registerInstrumentations({
+  instrumentations: [
+    getNodeAutoInstrumentations({
+      '@opentelemetry/instrumentation-http': {
+        startIncomingSpanHook: (req) => {
+          delete req.headers.traceparent
+          delete req.headers[`x-cloud-trace-context`]
+          delete req.headers[`grpc-trace-bin`]
+
+          return {}
+        },
+      },
+    }),
+  ],
+})
+
+const resource = Resource.default().merge(
+  new Resource({
+    [SemanticResourceAttributes.SERVICE_NAME]: process.env.TRACING_SERVICE_NAME,
+    [SemanticResourceAttributes.SERVICE_VERSION]: '0.1.0',
```

Comment:
is this supposed to be the version number of the combiner?

Reply:
It will appear like this in the traces `service.version`:
![Screenshot 2023-08-17 at 18 16 08](https://github.com/celo-org/celo-monorepo/assets/35505302/2e66d42a-f3b9-4e28-baed-2e95f41c258f).

It's an optional param, we can make it the version of the combiner if we want, remove it, or use a completely new numbering if we want. I'm OK with either option actually.